IDEAS.md

```diff
@@ -7,6 +7,8 @@ Some ideas could be done in a matter of hours, some could take months.
 ### Synchronize source code repository on p2p network
 Github is proprietary. If Gupaxx aims to be free software, it should not only be available on this proprietary platform and we should explore other options to be github free.  
 We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
+we can use [Codeberg](https://codeberg.org/) as an foss alternative to github
+we can create mirrors between github, codeberg and radicle.(doesnt have to be all 3)
```

Comment:
Codeberg would be better than github, but just only because Microsoft won't be in control and I would. I would prefer something like a ForgeFed (federated repo), where a contributor would not have to create an account on my forge instance. Over tor/i2p would be even better to not rely on domain names. 

But Radicle is just a better option overhaul since it's true p2p instead of federated. 
Making Forgefed, Radicle and Github synchronized would be quite a lot of work I think, so for now I prefer to think only about the synchronization of Radicle (for decentralization) and github (for lower barrier).
So for now, I will keep only Radicle in the ideas. Anyone is free to host Gupaxx on codeberg and sync with Github if he wants to.

Reply:
removed codeberg